docs/changehistory/NextVersion.md

```diff
@@ -6,13 +6,15 @@ publish: false
 
 Table of contents:
 
-- [Tracing API deprecation](#tracing-api-deprecation)
-- [Batched tileset enhancements](#batched-tileset-enhancements)
-  - [Per-model display settings](#per-model-display-settings)
-  - [Support for excluded models](#support-for-excluded-models)
-  - [Batch table property access](#batch-table-property-access)
-- [Geometry](#geometry)
-  - [Range tree search](#range-tree-search)
+- [NextVersion](#nextversion)
```

Comment:
drop this top level (i know it gets auto added by the markdown formatters)

Reply:
removed, should be good now

Reply:
this is back

Reply:
😮‍💨 my bad. fixed now